packages/web-components/package.json

```diff
@@ -39,6 +38,7 @@
   "devDependencies": {
     "@axe-core/puppeteer": "^4.10.0",
     "@babel/core": "^7.12.13",
+    "@department-of-veterans-affairs/css-library": "^0.13.1",
```

Comment:
This was originally in the `dependencies` list but I think because this repo is set up using workspaces, I could not update the USWDS package in the css-library without also needing it done in the web-components packages. Moving it to `devDependencies` fixed the build issue I was having.

We have a [separate ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3401) for updating css-library in other packages.